packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -120,6 +115,17 @@ describe("SharedString interval collections", () => {
 		);
 	});
 
+	function checkIntervalChange() {
+		const collection = sharedString.getIntervalCollection("test");
+		const interval = collection.add({ start: 0, end: 0 });
+		const intervalId = interval.getIntervalId();
+		assert(intervalId);
+		// @ts-expect-error - require passing both interval endpoints
+		collection.change(intervalId, { start: 1, end: undefined });
```

Comment:
it's a little silly to include this now since you backed out this portion of the PR. But if we're planning on doing it soon anyway, i'd rather have this lying around for a little bit than forget to add it later, so kinda on indifferent on removing it.

Reply:
I have a task for next sprint tracking the change cleanup, so I'll just leave it in.